test/test_ui.py

```diff
@@ -1136,7 +1137,12 @@ def test_summarize_items(self):
         summary = commands.summarize_items([self.item], False)
         self.assertEqual(summary, u"1 items, F, 4kbps, 10:54, 987.0 B")
 
-        i2 = deepcopy(self.item)
+        # make a copy of self.item
+        i2 = library.Item()
+        i2.bitrate = 4321
+        i2.length = 10 * 60 + 54
+        i2.format = "F"
```

Comment:
Any particular reason why this is not `i2 = self.item.copy()`?

Reply:
I wonder why this hasn't been done long ago, in a docstring of one of the parent classes it explicitly says that ```deepcopy()``` doesn't work on these objects.